config/tikv.toml

```diff
@@ -148,6 +148,7 @@ address = "pushgateway:9091"
 # bit smaller.
 # region-max-size = "144MB"
 # region-split-size = "96MB"
+region-split-size = "1MB"
```

Comment:
why do we do the change?

Reply:
as discussed yesterday

This PR sets TiKV region size from 96M to 1M, so that bugs are easier to appear.